roadmap/implementers-guide/src/node/utility/provisioner.md

```diff
@@ -42,12 +42,49 @@ Note that block authors must re-send a `ProvisionerMessage::RequestBlockAuthorsh
 
 When a validator is selected by BABE to author a block, it becomes a block producer. The provisioner is the subsystem best suited to choosing which specific backed candidates and availability bitfields should be assembled into the block. To engage this functionality, a `ProvisionerMessage::RequestInherentData` is sent; the response is a set of non-conflicting candidates and the appropriate bitfields. Non-conflicting means that there are never two distinct parachain candidates included for the same parachain and that new parachain candidates cannot be backed until the previous one either gets declared available or expired.
 
-Our goal with respect to the bitfields is simple: maximize availability. Always include all bitfields. It's a little more complicated with respect to candidates. The fundamental task there is to see which core the candidate can be included on. If its para is on a `ScheduledCore`, then it is unconditionally includable. Otherwise, we have two cases to deal with:
+### Bitfield Selection
 
-- If it is `next_up_on_available` on an `OccupiedCore`, we can only include the candidate if the bitfields we are including, merged with the `availability` vec of the `OccupiedCore`, together comprise 2/3+ of validators.
-- If it is `next_up_on_time_out` on an `OccupiedCore`, we can only include the candidate if the bitfields we are including, merged with the `availability` vec of the `OccupiedCore` together comprise <2/3 of validators, and the `time_out_at` is the block we are building.
+Our goal with respect to bitfields is simple: maximize availability. However, it's not quite as simple as always including all bitfields; there are constraints which still need to be met:
 
-Otherwise, the para isn't represented anywhere in the set of cores, and is therefore not scheduled, and can't be included.
+- We cannot choose more than one bitfield per validator.
+- Each bitfield must correspond to an occupied core.
+
+Beyond that, a semi-arbitrary selection policy is fine. In order to meet the goal of maximizing availability, a heuristic of picking the bitfield with the greatest number of 1 bits set in the event of conflict is useful.
+
+### Candidate Selection
+
+The goal of candidate selection is to determine which cores are available, and then to the degree possible, pick a candidate appropriate to each available core.
```

Comment:
I would phrase this instead as "which cores are free" since it doesn't overload the "available" terminology. Availability is a state transition for a core from occupied to free, as seen in the diagram here: https://w3f.github.io/parachain-implementers-guide/runtime/scheduler.html